src/assets/AssetsContractController.ts

```diff
@@ -1,10 +1,13 @@
 import BaseController, { BaseConfig, BaseState } from '../BaseController';
 
+// TODO: Destructuring this line introduces a compilation error
```

Comment:
:thinking:

Reply:
I guess TypeScript doesn't notice it until it's destructured—the BN type is used in a public-facing type but isn't exported itself, which makes it hard (impossible in some cases) for consumers of the package to get the types to work. This doesn't affect us right now since we don't use these types anywhere.